component/template/Cargo.toml

```
@@ -5,7 +5,9 @@ edition = "2021"
 description = "{{project-description}}"
 authors = ["{{authors}}"]
 build = "build.rs"
-{%- if repository != "" %}repository = "{{repository}}"{% endif %}
+{%- if repository != "" %}
+repository = "{{repository}}"
+{%- endif %}
```

```suggestion
{% if repository != "" -%}
repository = "{{repository}}"
{%- endif %}
```
Just a slight change to where the whitespace is added - the effect is the same though.